vllm/model_executor/models/bamba.py

```diff
@@ -277,6 +277,7 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
             config.hidden_size,
             org_num_embeddings=config.vocab_size,
         )
+        self.mamba2_metadata = None
```

Comment:
Could you please revert the changes to `bamba.py` I don't think the mamba2_metadata should be stored in `self`